src/runtime_src/core/tools/xbmgmt2/ReportPlatform.cpp

```diff
@@ -27,6 +27,7 @@ namespace XBU = XBUtilities;
 
 // 3rd Party Library - Include Files
 #include <boost/format.hpp>
+#include <boost/algorithm/hex.hpp>

```

Comment:
Forgot to remove this. I will remove this with any other changes that you request :)